app/Http/Controllers/Bible/StreamController.php

```diff
@@ -103,15 +103,15 @@ public function transportStream(Response $response, $fileset_id = null, $file_id
         $currentBandwidth->transportStream = sizeof($currentBandwidth->transportStreamBytes) ? $currentBandwidth->transportStreamBytes : $currentBandwidth->transportStreamTS;
 
         $current_file = "#EXTM3U\n";
-        $current_file .= '#EXT-X-TARGETDURATION:' . ceil($currentBandwidth->transportStream->max('runtime')) . "\n";
+        $current_file .= '#EXT-X-TARGETDURATION:' . ceil($currentBandwidth->transportStream->sum('runtime')) . "\n";
         $current_file .= "#EXT-X-VERSION:4\n";
         $current_file .= "#EXT-X-MEDIA-SEQUENCE:0\n";
         $current_file .= '#EXT-X-ALLOW-CACHE:YES';
 
         $signed_files = [];
 
         foreach ($currentBandwidth->transportStream as $stream) {
-            $current_file .= "\n#EXTINF:$stream->runtime,";
+            $current_file .= "\n#EXTINF:$stream->runtime";
```

Comment:
https://tools.ietf.org/html/rfc8216#section-4.3.2.1 indicates that a comma is necessary, and title is optional.  why does removing the comma (and title) fix the issue of truncated playback?

Reply:
Reverted this change Jon

Reply:
thanks!